sklearn/multiclass.py

```diff
@@ -160,7 +160,9 @@ class OneVsRestClassifier(MultiOutputMixin, ClassifierMixin,
         :term:`decision_function` or :term:`predict_proba`.
 
     n_jobs : int or None, optional (default=None)
-        The number of jobs to use for the computation.
+        The number of jobs to use for the computation: the n_classes
```

Comment:
```suggestion
        The number of jobs to use for the computation: the `n_classes`
```

Reply:
Hey @NicolasHug, I have modified accordingly. Can you please check the PR? Thanks!